src/test/java/org/candlepin/resource/UserResourceTest.java

```diff
@@ -329,7 +330,7 @@ public void testUpdateUserChangePassword() {
 
         user = this.userCurator.get(user.getId());
         assertNotNull(user);
-        assertEquals(Util.hash("Skywalker"), user.getPassword());
+        assertTrue(OpenBSDBCrypt.checkPassword(user.getPassword(), "Skywalker".toCharArray()));
```

Comment:
Ditto on this. We shouldn't be using BCrypt outside of the default user service adapter or its tests.